<?xml version="1.0"?>
<launch>
  <!-- Input/Output Topics: -->
  <arg name="point_cloud" default="cam3d/depth_registered/points"/>
  <arg name="shape_array" default="supporting_plane_extraction/shape_array_pub"/>
  <arg name="cluster_array" default="tabletop_object_cluster/cluster_array"/>
  <arg name="bounding_box_array" default="tabletop_object_cluster/bounding_box_array"/>
  <arg name="marker_bb" default="bb_marker"/>

  <!-- start environment modelling -->
  <!--node pkg="nodelet" type="nodelet" name="cam3d_nodelet_manager"  args="manager" output="screen"/-->


  <node pkg="cob_table_object_cluster" type="table_object_cluster_node" name="table_object_cluster" output="screen" ><!--launch-prefix="gdb -ex run -args"-->
    <remap from="/get_point_map" to="/point_map/get_map"/>

    <remap from="point_cloud" to="$(arg point_cloud)"/>
    <remap from="shape_array" to="$(arg shape_array)"/>
    <remap from="cluster_array" to="$(arg cluster_array)"/>
    <remap from="bounding_box_array" to="$(arg bounding_box_array)"/>
    <remap from="marker_bb" to="$(arg marker_bb)"/>

    <param name="file_path" type="string" value="/home/goa/tmp/"/>
    <param name="save_to_file" type="bool" value="false"/>
    <!--param name="height_min" type="double" value="0.03"/>
    <param name="height_max" type="double" value="0.5"/-->
    <param name="height_min" type="double" value="-0.5"/>
    <param name="height_max" type="double" value="-0.03"/>
    <param name="min_cluster_size" type="int" value="10"/>
    <param name="cluster_tolerance" type="double" value="0.06"/>
    <param name="enable_action_mode" type="bool" value="false"/>
  </node>


</launch>

